### PR TITLE
all app sites

### PR DESCRIPTION
**Pull request by RBN2208**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,24 +2,40 @@ import Grid from './components/Grid'
 import AppHeader from './components/AppHeader'
 import getCharacters from './services/getCharacters'
 import createElement from './lib/createElement'
-import Card from './components/Card'
+// import Card from './components/Card'
```
Comment (macfrei) [inline src/App.js:5]: If you don't use the component, then please delete this comment. We always want to main branch to be nice and clean.

File: src/App.js [hunk 2]
```diff
@@ -2,24 +2,40 @@ import Grid from './components/Grid'
...
 import Homepage from './components/Homepage'
+import PeoplePage from './components/PeoplePage'
+import MapPage from './components/MapPage'
+import HousePage from './components/HousePage'
+import CreatePage from './components/CreatePage'
+// import Navigation from './components/Navigation'
```
Comment (macfrei) [inline src/App.js:11]: Same here.

File: src/App.js [hunk 3]
```diff
@@ -2,24 +2,40 @@ import Grid from './components/Grid'
...
+  )
   document.body.append(grid)
 
   // fetch API
   getCharacters()
     .then(characters => createCards(characters))
     .catch(error => handleGetCharacterError(error))
 
+  // get characters to peoplepage
   function createCards(characters) {
```
Comment (macfrei) [inline src/App.js:37]: If this function doesn't create cards, then maybe think about renaming the function? Then you also wouldn't need to comment in line 36.

File: src/components/Homebutton/Homebutton.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import createElement from '../../lib/createElement'
+import './Homebutton.css'
+
+import homeSVG from '@fortawesome/fontawesome-free/svgs/solid/home.svg'
+
+const buttonConfig = [{ svgPath: homeSVG, text: 'Home' }]
+
+export default function Homebutton(onNavigate) {
+  const buttons = buttonConfig.map(({ svgPath, text }) => {
```
Comment (macfrei) [inline src/components/Homebutton/Homebutton.js:9]: You might want to consider this as you only have one object in your `buttonConfig`. If you call it `buttons` I would think that there are multiple buttons created, however, it is just one.

**Review by macfrei — COMMENTED** [review]
Nice job! I added a few comments, if you have any question, please get back to me! 🚀

File: src/App.js (hunk 1 above)
Comment (RBN2208) [reply src/App.js:5]: yeah, already deleted! i comment working code when i want to rewrite the code, so i dont need to press "command+z" so often when i dont get a new working code^^

**Pull request merged**